sklearn/metrics/scorer.py

```diff
@@ -514,6 +514,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 log_loss_scorer = make_scorer(log_loss, greater_is_better=False,
                               needs_proba=True)
 log_loss_scorer._deprecation_msg = deprecation_msg
+brier_score_loss_scorer = make_scorer(brier_score_loss)
```

Comment:
Hey @qinhanmin2014, I think that the `greater_is_better` parameter in `make_scorer` should be set to `False` because `brier_score_loss` is a loss function. It looks like it is `True` by default (see https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/metrics/scorer.py#L400)